ddl/reorg.go

```diff
@@ -234,7 +234,7 @@ func (w *worker) runReorgJob(rh *reorgHandler, reorgInfo *reorgInfo, tblInfo *mo
 			return dbterror.ErrCancelledDDLJob
 		}
 		rowCount, _, _ := rc.getRowCountAndKey()
-		logutil.BgLogger().Info("[ddl] run reorg job done", zap.Int64("handled rows", rowCount))
+		logutil.BgLogger().Info("[ddl] run reorg job done", zap.Int64("handled rows", rowCount), zap.Error(err))
```

Comment:
it contains err message. I think the log level is not info.

Reply:
It should be like this. 

```
if err != nil {
     log.Error(....)
} else {
     log.Info(....)
}
```

Reply:
The error can be `nil`.

Reply:
OK. Let's improve it in master.